ui/client/components/Process/ProcessInfo.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react"
+import {ProcessStateType, ProcessType} from "./ProcessTypes"
+import {
+  descriptionProcessArchived,
+  descriptionSubprocess,
+  descriptionSubprocessArchived,
+  unknownDescription,
+} from "./ProcessMessages"
+import {CSSTransition, SwitchTransition} from "react-transition-group"
+import ProcessStateIcon, {unknownIcon} from "./ProcessStateIcon"
+import {absoluteBePath} from "../../common/UrlUtils"
+
+type State = {}
+
+type OwnProps = {
+  processState?: ProcessStateType,
+  isStateLoaded: boolean,
+  process: ProcessType,
+  iconHeight: number,
+  iconWidth: number,
+}
+
+class ProcessInfo extends React.Component<OwnProps, State> {
+  static defaultProps = {
+    isStateLoaded: false,
+    processState: null,
+    iconHeight: 32,
+    iconWidth: 32,
+  }
+
+  static subprocessIcon = "/assets/process/subprocess.svg"
+  static archivedIcon = "/assets/process/archived.svg"
+
+  private animationTimeout = {
+    enter: 500,
+    appear: 500,
+    exit: 500,
+  }
+
+  private animationListener = (node, done) => node.addEventListener("transitionend", done, false)
+
+  private getDescription = (process: ProcessType, processState: ProcessStateType, isStateLoaded: boolean): string =>
```

Comment:
Please add some little TODO, that logic from `getDescription` and `getProcessIcon` should be moved to BE